apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx

```diff
@@ -144,9 +150,109 @@ export const ScriptBasedFlow: FunctionComponent<AdaptiveScriptsPropsInterface> =
     const [ showScriptResetWarning, setShowScriptResetWarning ] = useState<boolean>(false);
     const [ showConditionalAuthContent, setShowConditionalAuthContent ] = useState<boolean>(isMinimized);
     const [ isEditorFullScreen, setIsEditorFullScreen ] = useState<boolean>(false);
+    const [ showAddSecretModal, setShowAddSecretModal ] = useState<boolean>(false);
+    const [ editorInstance, setEditorInstance ] = useState(undefined);
```

Comment:
Check the proper type from the `react-components`.
```suggestion
    const [ editorInstance, setEditorInstance ] = useState<codemirror.Editor>(undefined);
```